### PR TITLE
Add pre-commit install instructions

### PR DESCRIPTION
**Pull request by jooola**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -20,7 +23,7 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/savonet/pre-commit-ocamlformat
-    rev: v1
```
Comment (toots) [inline .pre-commit-config.yaml:23]: Any reason to revert this? Commit tags are more user-friendly.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+#
+# Run the following command to set up the pre-commit git hook scripts:
+# $ pre-commit install
```
Comment (toots) [inline .pre-commit-config.yaml:6]: Ultimately, we should start writing a contributor/developper's guide. But this is good for now.

**Review by toots — COMMENTED** [review]
This looks good. Just one question about the change in of `rev` for the `ocamlformat` check.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/savonet/pre-commit-ocamlformat
-    rev: v1
+    rev: d120e7d5a
```
[the remark this answers is not in the document]
Comment (jooola) [reply .pre-commit-config.yaml:26]: I think the v1 is understood as v1.x.x, using a  tag like v1.0.0 should remove the warning.

**Comment by toots** [comment]
I forked the github action repository and added `pre-commit` setup and opam repository entries for our internal packages over at: https://github.com/savonet/liquidsoap-dev-tools. I think that this could act at the new `liquidsoap-full` for all builds.

**Pull request merged**